server/src/main/java/com/objectcomputing/checkins/services/reviews/ReviewPeriodServicesImpl.java

```diff
@@ -21,14 +32,34 @@ class ReviewPeriodServicesImpl implements ReviewPeriodServices {
     private static final Logger LOG = LoggerFactory.getLogger(ReviewPeriodServicesImpl.class);
 
     private final ReviewPeriodRepository reviewPeriodRepository;
+    private final ReviewAssignmentRepository reviewAssignmentRepository;
+    private final MemberProfileRepository memberProfileRepository;
     private final ReviewStatusTransitionValidator reviewStatusTransitionValidator;
-
-    ReviewPeriodServicesImpl(
-            ReviewPeriodRepository reviewPeriodRepository,
-            ReviewStatusTransitionValidator reviewStatusTransitionValidator
-    ) {
+    private EmailSender emailSender;
+    private final Environment environment;
+    private final String webAddress;
+    public static final String WEB_ADDRESS = "check-ins.web-address";
+
+    @Value("${aes.key}")
+    private String aesKey;
+
+    public ReviewPeriodServicesImpl(ReviewPeriodRepository reviewPeriodRepository,
+                                    ReviewAssignmentRepository reviewAssignmentRepository,
+                                    MemberProfileRepository memberProfileRepository,
+                                    CurrentUserServices currentUserServices,
+                                    @Named(MailJetConfig.HTML_FORMAT) EmailSender emailSender,
+                                    Environment environment, @Property(name = WEB_ADDRESS) String webAddress) {
         this.reviewPeriodRepository = reviewPeriodRepository;
+        this.reviewAssignmentRepository = reviewAssignmentRepository;
+        this.memberProfileRepository = memberProfileRepository;
         this.reviewStatusTransitionValidator = reviewStatusTransitionValidator;
+        this.emailSender = emailSender;
+        this.environment = environment;
+        this.webAddress = webAddress;
+    }
+
+    public void setEmailSender(EmailSender emailSender) {
```

Comment:
This doesn't need to be public as it's only used in test

```suggestion
   void setEmailSender(EmailSender emailSender) {
```